compiler/rustc_borrowck/src/region_infer/opaque_types.rs

```diff
@@ -418,6 +417,7 @@ fn check_opaque_type_parameter_valid(
                 .into_iter()
                 .map(|i| tcx.def_span(opaque_generics.param_at(i, tcx).def_id))
                 .collect();
+            //FIXME: wait for lists? I tried make Vec<Span> into MutiSpan.
```

Comment:
```suggestion
            // FIXME(#100717) needs eager translation/list support
```
if you need to leave a comment like this :)